docs/supported_formats/hspy.rst

```diff
@@ -3,43 +3,44 @@
 HSpy - HyperSpy's HDF5 Specification
 ------------------------------------
 
-This is `HyperSpy's <https://hyperspy.org>`_ default format and it is the only one that guarantees that no
+This is `HyperSpy's <https://hyperspy.org>`_ default format and for data processed
+in HyperSpy, it is the only format that guarantees that no
 information will be lost in the writing process and that supports saving data
 of arbitrary dimensions. It is based on the `HDF5 open standard
 <https://www.hdfgroup.org/solutions/hdf5/>`_. The HDF5 file format is supported by `many
 applications
 <https://support.hdfgroup.org/products/hdf5_tools/SWSummarybyName.htm>`_.
-Part of the specification is documented in :external+hyperspy:ref:`metadata_structure`.
+Parts of the specifications are documented in :external+hyperspy:ref:`metadata_structure`.
 
-.. versionadded:: 1.2
+.. versionadded:: HyperSpy_v1.2
```

Comment:
Is there a reason for the underscore?
```suggestion
.. versionadded:: HyperSpy v1.2
```

Reply:
Yes there is, only the first word after `.. versionadded::` is printed bold (or even printed in the header, I don't quite remember). Without `HyperSpy` it seems like we are referrinf to RosettaSciIO versions and without the underscore it is not clear, which HyperSpy version one refers to. Any alternative symbol would of course be fine.

Reply:
ok, this sounds the best, then!